src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.ValidateToken.cs

```diff
@@ -595,7 +596,23 @@ internal async ValueTask<TokenValidationResult> ValidateTokenPayloadAsync(
                 if (!tokenValidationResult.IsValid)
                     return tokenValidationResult;
             }
+            if (AppContextSwitches.SerializeDeserializeActorClaim && jsonWebToken.TryGetPayloadValue<JsonElement>(validationParameters.ActorClaimName, out var actClaim))
```

Comment:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/9a7b3b70348ff9850163e42981b80c76caab9ba2/src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.ClaimsIdentity.cs#L68C17-L81C18 could be a good place to delegate validation checks and claims identity creation for ActorClaimType.